server/src/main/java/io/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -68,6 +68,8 @@
   private static final String HOST_ATTRIBUTE = "io.druid.proxy.to.host";
   private static final String QUERY_ATTRIBUTE = "io.druid.proxy.query";
   private static final String OBJECTMAPPER_ATTRIBUTE = "io.druid.proxy.objectMapper";
+  public static final int CANCELLATION_TIMEOUT = 500;
+  public static final int MAX_QUEUED_CANCELLATIONS = 64;
```

Comment:
requests only get queued of all connections to that host are busy. the number of connections by default is already 200